asgardio.Dockerfile

```diff
@@ -0,0 +1,275 @@
+    && unzip -o /build/software/jce/jce_policy-8.zip -d /build/software/jce \
+    && mv /build/software/java/jdk1.8.0_221/jre/lib/security/policy/unlimited/local_policy.jar /build/software/java/jdk1.8.0_221/jre/lib/security/policy/unlimited/local_policy-original.jar \
+    && mv /build/software/java/jdk1.8.0_221/jre/lib/security/policy/unlimited/US_export_policy.jar /build/software/java/jdk1.8.0_221/jre/lib/security/policy/unlimited/US_export_policy-original.jar \
+    && cp /build/software/jce/UnlimitedJCEPolicyJDK8/local_policy.jar /build/software/java/jdk1.8.0_221/jre/lib/security/policy/unlimited/ \
+    && cp /build/software/jce/UnlimitedJCEPolicyJDK8/US_export_policy.jar /build/software/java/jdk1.8.0_221/jre/lib/security/policy/unlimited/ \
+    && rm  /build/software/jce/jce_policy-8.zip \
+    && rm -r /build/software/jce/UnlimitedJCEPolicyJDK8
+
+ENV JAVA_HOME /build/software/java/jdk1.8.0_221
+
+RUN \
+    wget -P /build/software/android https://dl.google.com/android/repository/sdk-tools-linux-4333796.zip \
+    && unzip /build/software/android/sdk-tools-linux-4333796.zip -d /build/software/android/ \
+    && rm /build/software/android/sdk-tools-linux-4333796.zip \
+    && yes | /build/software/android/tools/bin/sdkmanager --licenses \
+    && chmod 777 -R /build/software/android/ \
+    && unset JAVA_HOME
+
+ENV ANDROID_HOME /build/software/android
+
+RUN rm /usr/bin/java \
+    && ln -s /usr/lib/jvm/java-1.8.0-openjdk-amd64/bin/java  /usr/bin/java
+
+RUN \
+    echo "net.ipv4.ip_local_port_range=15000 61000" >> /etc/sysctl.conf \
+    && echo "net.ipv4.tcp_fin_timeout=30" >> /etc/sysctl.conf \
+    && echo "*	soft	nofile	65535" >> /etc/security/limits.conf \
+    && echo "*	hard	nofile	65535" >> /etc/security/limits.conf \
+    && echo "*	soft	nproc	65535" >> /etc/security/limits.conf \
+    && echo "*	hard	nproc	65535" >> /etc/security/limits.conf
+
+RUN \
+    mkdir -p /home/jenkins
+
+RUN mkdir -p /build/gpg-keys/.gnupg
+ADD .gnupg /build/gpg-keys/.gnupg
+
+ARG JENKINS_REMOTING_VERSION=3.5
+
+# See https://github.com/jenkinsci/docker-slave/blob/2.62/Dockerfile#L32
+RUN curl --create-dirs -sSLo /usr/share/jenkins/slave.jar https://repo.jenkins-ci.org/public/org/jenkins-ci/main/remoting/$JENKINS_REMOTING_VERSION/remoting-$JENKINS_REMOTING_VERSION.jar \
+  && chmod 755 /usr/share/jenkins \
+  && chmod 644 /usr/share/jenkins/slave.jar \
+  && chmod a+rwx /home/jenkins
+
+# Install Docker Engine
+RUN apt-get update && \
+apt-get -y install apt-transport-https \
+     ca-certificates \
+     curl \
+     gnupg2 \
+     software-properties-common && \
+curl -fsSL https://download.docker.com/linux/$(. /etc/os-release; echo "$ID")/gpg > /tmp/dkey; apt-key add /tmp/dkey && \
+add-apt-repository \
+   "deb [arch=amd64] https://download.docker.com/linux/$(. /etc/os-release; echo "$ID") \
+   $(lsb_release -cs) \
+   stable" && \
+apt-get update && \
```

Comment:
@gayashanbc please align these two lines.

Reply:
Resolved in https://github.com/wso2/build-docker/pull/17/commits/666581e7c8f3bbe21071723838b795226b882441